spk/python3/Makefile

```diff
@@ -21,7 +21,7 @@ DESCRIPTION_SPN = Lenguaje de programación Python.
 RELOAD_UI = yes
 STARTABLE = no
 DISPLAY_NAME = Python3
-CHANGELOG = "Update to Python 3.6.8"
+CHANGELOG = "Fixed cross compile settings"
```

Comment:
Since this will be the next published version (previous was retracted), I would suggest keeping both the bump and the cross compile fix in the changelog.